apps/api/src/domain/models/song/commands/add-lyrics-for-song/add-lyrics-for-song.command-handler.ts

```diff
@@ -0,0 +1,7 @@
+import { CommandHandler } from '@coscrad/commands';
+import { BaseCommandHandler } from '../../../shared/command-handlers/base-command-handler';
+import { Song } from '../../song.entity';
+import { AddLyricsForSong } from './add-lyrics-for-song.command';
+
+@CommandHandler(AddLyricsForSong)
+export class AddLyricsForSongCommandHandler extends BaseCommandHandler<Song> {}
```

Comment:
Once you have the happy-path (valid) test case working, running the test should fail because the handler is missing its implementation. At that point, you'll have to implement all the methods that are required by the base class.